CHANGELOG.md

```diff
@@ -9,7 +9,7 @@
 * update unstoppable domains package [#3352](https://github.com/MyEtherWallet/MyEtherWallet/pull/3352)
 
 ### fix
-
+* fix access wallet side padding [#3374](https://github.com/MyEtherWallet/MyEtherWallet/pull/3374)
```

Comment:
Don't need to do this anymore. Please remove

Reply:
I removed it.